### PR TITLE
multiple docs styling fixes and few typos;

### PR DESCRIPTION
**Pull request by popravich**

### CODE REVIEW

File: docs/build_commands.rst [hunk 1]
```diff
@@ -368,6 +368,19 @@ The following ``pip`` package specification formats are supported:
...
+     to be able to install ``git+/hg+`` python packages:
+
+     .. code-block:: yaml
+
+        setup:
+        - !Ubuntu trusty
+        - !Install
+          - ca-certificates
+        - !Py2Install
+          - "git+https://github.com/KeepSafe/aiohttp#egg=aiohttp"
```
Comment (popravich) [inline docs/build_commands.rst]: a bit of ads)

File: docs/build_commands.rst [hunk 2]
```diff
@@ -368,6 +368,19 @@ The following ``pip`` package specification formats are supported:
 * The ``git+`` and ``hg+`` links (the git and mercurial are installed as build
   dependency automatically)
 
+  .. note::
+     In Ubuntu you may also need to install ``ca-certificates`` package
+     to be able to install ``git+/hg+`` python packages:
```
Comment (tailhook) [inline docs/build_commands.rst]: Isn't it just a bug?

Comment (popravich) [inline docs/build_commands.rst]: Not sure who's bug this is.
In ubuntu `pip` exits with following error:
`Problem with the SSL CA cert (path? access rights?)`

Comment (tailhook) [inline docs/build_commands.rst]: Yes. We install git and hg automatically. Should probably add `ca-certificates` to the list (#58)

Comment (popravich) [inline docs/build_commands.rst]: sounds reasonable

**Pull request merged**